Makefile

```diff
@@ -0,0 +1,8 @@
+all:
+	virtualenv .test-ve
```

Comment:
It might be good to specify the exact major version of python so we don't end up using Python 3.